src/org/labkey/test/components/glassLibrary/grids/ResponsiveGrid.java

```diff
@@ -141,6 +141,18 @@ public ResponsiveGrid selectRow(int index, boolean checked)
         return getThis();
     }
 
+    /**
+     * Finds the first row with the specified texts in the specified columns, and sets its checkbox
+     * @param partialMap    key-column, value-text in that column
+     * @param checked       the desired checkbox state
+     * @return
+     */
+    public ResponsiveGrid selectRow(Map partialMap, boolean checked)
```

Comment:
You should include the generic types for the Map.

```suggestion
    public ResponsiveGrid selectRow(Map<String, String> partialMap, boolean checked)
```

Also the `selectRow(String columnLabel, String text, boolean checked)` method validates the checkbox count after selecting.  Do you need to do that here too?

Reply:
It's probably a good idea so I will